examples/visualization/coregistration.py

```diff
@@ -0,0 +1,71 @@
+"""
+=========================================
+Use automated approach to co-registration
```

Comment:
@drammock it would be good if you could comment on where this should live, and maybe push some commits to restructure to have a more narrative form.

Also I think that the functions that the `fit_*` functions should `logger.info` the dig<->MRI distances before the fit and after the fit. Then they can just be called with `verbose=True` in the tutorial in separate code blocks (`# %%`) along with a `plot_alignment` call after each step), and then the outputs will be nice and readable in the SG output.

Reply:
I feel like `examples/visualization` is not the right place for this, as the point of coreg is mostly about finding the right transform, and the visualization is just the tool we use to do that. IMO there isn't really an appropriate section within `examples`...  it would fit pretty naturally as `tutorials/forward/25_automated_coreg.py` (coming right after the tutorial about general coreg / coord frame concepts).

To fit in as a tutorial it would need a more verbose / extended narrative, but wouldn't need to be *hugely* longer.  I'd recommend moving it to that location now, and expand the narrative either now (if @jhouck wants to expand it) or in a separate PR (I can do it if @jhouck doesn't want to).


> Also I think that the functions that the `fit_*` functions should `logger.info` the dig<->MRI distances before the fit and after the fit. Then they can just be called with `verbose=True` in the tutorial in separate code blocks (`# %%`) along with a `plot_alignment` call after each step), and then the outputs will be nice and readable in the SG output.

+1 for this suggestion; I often think that if we're `print`ing something in a tutorial then maybe it's something we ought to be logging anyway.

Reply:
I moved the tutorial and I added `_log_dig_mri_distance()`.